doc/cephadm/services/monitoring.rst

```diff
@@ -344,10 +344,13 @@ Setting up Prometheus
 Setting Prometheus Retention Time
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Cephadm provides the option to set the Prometheus TDSB retention time using
-a ``retention_time`` field in the Prometheus service spec. The value defaults
-to 15 days (15d). If you would like a different value, such as 1 year (1y) you
-can apply a service spec similar to:
+Cephadm provides the option to set the Prometheus TDSB retention time specifying
+``retention_time`` and/or ``retention_size`` fields in the Prometheus service spec.
```

Comment:
```suggestion
Cephadm provides the option to set the Prometheus TDSB retention time and/or size by specifying
the ``retention_time`` and``retention_size`` fields respectively in the Prometheus service spec.
```
Personal opinion on wording. Feel free to just mark this one resolved if you disagree.

Reply:
I'd suggest something more concise like

 "Cephadm can configure Prometheus TSDB retention by specifying ``retention_time`` and``retention_size`` values in the Prometheus service spec."

Reply:
> I'd suggest something more concise like
> 
> "Cephadm can configure Prometheus TSDB retention by specifying `retention_time` and`retention_size` values in the Prometheus service spec."

I like that better as well. @rkachach if you're going to change I'd say go with @anthonyeleven suggestion over mine.

Reply:
fixed